docs/codelabs/building-beautiful-sites/docsite/6-wrapping-up.md

```diff
@@ -14,11 +14,12 @@ edited for a non-technical audience
 
 # Wrapping up
 
-By using some simple markup, ~100 lines of CSS, ~4 lines of JavaScript, and the Material Components for the web library, you have the start of a beautiful e-commerce app that conforms to the Material Design guidelines and looks great across all devices. Your designer is happy, and you get to leave work early.
+By using some simple markup, ~100 lines of CSS, ~4 lines of JavaScript, and the Material Components for the web library, you have the start of a beautiful e-commerce app that conforms to the Material Design guidelines and looks great across all devices.
 
-> A final version of the app is completed for you within the `docs/codelabs/building-beautiful-sites/completed directory`. Feel free to test your version of the site against the app in that directory. Don't forget to run npm install in that directory first!
-{: .hint}
+## Material Components
 
-## Next steps
+This was a quick example with just a few components but Material Components has so much more it can do on all platforms! Have your engineers go to material.io/components to see the amazing open-source code available for web, iOS, and Android.
```

Comment:
Might intended to have material.io/components as a link?

This was a quick example with just a few components but Material Components has so much more it can do on all platforms! Have your engineers go to [material.io/components](material.io/components) to see the amazing open-source code available for web, iOS, and Android.

Reply:
Done